src/api.jl

```diff
@@ -772,13 +769,21 @@ function load_json(ctx::Context, database::AbstractString, engine::AbstractStrin
     return exec(ctx, database, engine, source; inputs = inputs, readonly = false, kw...)
 end
 
-function load_model(ctx::Context, database::AbstractString, engine::AbstractString, models::Dict; kw...)
-    tx = Transaction(ctx.region, database, engine, "OPEN"; readonly = false)
-    actions = [_make_load_model_action(name, model) for (name, model) in models]
-    return _post(ctx, PATH_TRANSACTION; query = query(tx), body = body(tx, actions...), kw...)
+function load_models(ctx::Context, database::AbstractString, engine::AbstractString, models::Dict; kw...)
+    queries = [
+        """ def insert:rel:catalog:model[$(_escape_string_for_rel(name))] = $(_escape_string_for_rel(models[name])) """
+        for name in keys(models)
+    ]
```

Comment:
```suggestion
    queries = [
        """ def insert:rel:catalog:model[$(_escape_string_for_rel(name))] = $(_escape_string_for_rel(value)) """
        for (name,value) in models
    ]
```
Slightly cleaner iteration.